mmv1/third_party/tgc_next/caiasset/asset.go

```diff
@@ -0,0 +1,133 @@
+package caiasset
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+// Asset is the CAI representation of a resource.
+type Asset struct {
```

Comment:
Is this going to represent the CAI asset of any particular resource?

Reply:
It the general CAI asset for any resources and match with the [API Asset object](https://cloud.google.com/asset-inventory/docs/reference/rest/v1/Asset) mostly.